logstash-core/build.gradle

```diff
@@ -105,7 +105,7 @@ tasks.register("javaTests", Test) {
 tasks.register("rubyTests", Test) {
     inputs.files fileTree("${projectDir}/lib")
     inputs.files fileTree("${projectDir}/spec")
-    systemProperty 'logstash.core.root.dir', projectDir.absolutePath
+    systemProperty 'logstash.root.dir', projectDir.parent
```

Comment:
a new convention for all RSpec Java invocations ...
also `systemProperty 'logstash.core.root.dir'` wasn't previously always pointing to *logstash-core* 
(e.g. from *x-pack/build.gradle*)